tools/get.py

```diff
@@ -254,5 +425,11 @@ def identify_platform():
         if is_test:
             print("Would install: {0}".format(tool["archiveFileName"]))
         else:
-            get_tool(tool)
+            if not get_tool(tool, force_download, force_extract):
+                if verbose:
+                    print(f"Tool {tool['archiveFileName']} was corrupted. Re-downloading...\n")
+                if not get_tool(
+                    tool, True, force_extract
+                ):  # Corrupted file was renamed, will not be found and will be re-downloaded
+                    print(f"Tool {tool['archiveFileName']} was corrupted, but re-downloading did not help!\n")
```

Comment:
Maybe add an `sys.exit(1)` here so the CI will fail when downloading the libs fail

Reply:
go ahead

Reply:
I will. It was mostly a reminder for me to do it when I am back home